exercises/flatten-array/flatten-array_spec.lua

```diff
@@ -33,4 +33,11 @@ describe('flatten-array', function()
       flatten({ { { {  } } }, { { } } })
     )
   end)
+
+  it('should flatten array from readme example', function()
+    assert.are.same(
+      {1,2,3,4,5},
+      flatten({1,{2,3,nil,4},{null},5})
```

Comment:
This was actually omitted because it relies un undefined behavior. The `#` operator is only defined for arrays/tables that are sequences where a sequence is defined as a table where all of the positive integer keys (1-n) are non-nil. If you try to use `ipairs` or `#` on one of these objects, you can get any length up to an including what you'd consider to be the "correct" length. So, while this may work for this version of Lua, this is not guaranteed to work for any Lua implementation/version.